lib/macho/macho_file.rb

```diff
@@ -264,6 +264,60 @@ def change_rpath(old_path, new_path)
       set_path_in_rpath(rpath_cmd, old_path, new_path)
     end
 
+    # Add the given runtime path to the Mach-O.
+    # @example
+    #  file.rpaths # => ["/lib"]
+    #  file.add_rpath("/usr/lib")
+    #  file.rpaths # => ["/lib", "/usr/lib"]
+    # @param path [String] the new runtime path
+    # @return [void]
+    # @raise [MachO::RpathExistsError] if the runtime path already exists
+    def add_rpath(path)
+      raise UnimplementedError.new("addition of rpaths")
+      raise RpathExistsError.new(path) if rpaths.include?(path)
+    end
+
+    # Delete the given runtime path from the Mach-O.
+    # @example
+    #  file.rpaths # => ["/lib"]
+    #  file.delete_rpath("/lib")
+    #  file.rpaths # => []
+    # @param path [String] the runtime path to delete
+    # @return void
+    # @raise [MachO::RpathUnknownError] if no such runtime path exists
+    def delete_rpath(path)
+      rpath_cmds = command(:LC_RPATH).select { |r| r.path.to_s == path }
+      raise RpathUnknownError.new(old_path) if rpath_cmds.empty?
+
+      # just the reverse! should suffice since load commands are read in
+      # sequential order, but let's not rely on this behavior
+      rpath_cmds.sort_by! { |lc| lc.view.offset }.reverse!
```

Comment:
You can drop the `reverse!` here and compensate for this by using `reverse_each` in the iteration below.